### DISCUSSION TITLE
Pass data that is not in path variables when using router.push()

### DISCUSSION THREAD

**Discussion by ttc0419** [vuejs/router#2399] [Help and Questions]
https://github.com/vuejs/router/blob/main/packages/router/CHANGELOG.md#414-2022-08-22 removes the ability to pass data not in path.  But for my breadcrumb widget, it need both the resource name to display and resource id to let rest client to fetch the data.  And it renders itself base on the matched routes, I don't see a clean way to do it other than include both label and id as path variables, but it would be very redundant. Any idea?

**Comment by Deckluhm** [comment]
Did you try any alternative suggested in the changelog?

Since it has been removed because it was an anti-pattern, you better try to do it another way.

If you give a bit more info about your code (ideally just a minimal example of your code) I can help suggest something.
